service/controller/resource/alerting/alertmanagerconfig/resource_test.go

```diff
@@ -110,7 +110,53 @@ func TestRenderingOfAlertmanagerConfig(t *testing.T) {
 	}
 
 	for _, flavor := range unittest.ProviderFlavors {
-		outputDir, err := filepath.Abs("./test/alertmanager-config/" + flavor)
+		outputDir, err := filepath.Abs("./test/alertmanager-config/classic/" + flavor)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		c := unittest.Config{
+			OutputDir:            outputDir,
+			T:                    t,
+			TestFunc:             testFunc,
+			TestFuncReturnsBytes: true,
+			Flavor:               flavor,
+			Update:               *update,
+		}
+		runner, err := unittest.NewRunner(c)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		err = runner.Run()
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
+}
+
+func TestRenderingOfAlertmanagerConfigWithMimirEnabled(t *testing.T) {
```

Comment:
This add new tests to be able to render alertmanager config when mimir is enabled